public/app/screens/main/sections/devices/style.scss

```diff
@@ -4,6 +4,8 @@
   padding: $global-padding;
 
   .cc-devices__button {
-    @include button()
+    @include button();
+    // TODO: Use variable: $column-height
```

Comment:
The padding before the button depends on the column height. I don't want to hard code 50px here  as we otherwise would loose this semantic information. It's not yet used because this variable is introduced by another PR: https://github.com/yetu/controlcenter/pull/47